### ISSUE TITLE
Vulnerable to XSS attack

### ISSUE THREAD

**Issue by balexand** [imgix/imgix-rails#21]
Using the following source:

```erb
<% source = '/foo"><script>alert("hacked")</script><' %>
```

Either of the following will result in an XSS attack:

```erb
<%= ix_image_tag source %>
```

```erb
<%= image_tag ix_image_url source %>
```

You can verify this by pasting the code into any Rails ERB template. The browser will show an alert dialog saying "hacked".

The source of the vulnerability is [this line](https://github.com/imgix/imgix-rails/blob/01d147ab292e3d471f7ae7a60a0532aa06b7ca33/lib/imgix/rails/url_helper.rb#L11). `html_safe` is being called without verifying that the input is safe. Here's the [documentation for `html_safe`](http://api.rubyonrails.org/classes/String.html#method-i-html_safe).

**Comment by paulstraw** [comment]
Hey @balexand, thanks for the bug report. This is something we're aware of and are going to be tackling next week. It is effectively a symptom of some known encoding issues present in [imgix/imgix-rails](https://github.com/imgix/imgix-rails),  which is what generates the URLs for imgix-rails. I'll definitely keep you in the loop on this as it progresses on our end.

**Comment by paulstraw** [comment]
Alrighty, this is fixed up by #22, which I'll be pushing out in a new release soon.
